src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private final Object[] keys = new Object[MAX_STORAGE_SIZE];
+    private final Object[] values = new Object[MAX_STORAGE_SIZE];
```

Comment:
```suggestion
    private final K[] keys;
    private final V[] values;
    ....
    
    public StorageImpl() {
        keys = (K[]) new Object[MAX_STORAGE_SIZE];
        values = (V[]) new Object[MAX_STORAGE_SIZE];
    }
```
it is better to move initialization to constructor

Reply:
Thank you, I'll keep it on mind